content/en/security/vulnerability_pipeline/mute.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Mute Rules
+---
+
+{{< callout url="https://www.datadoghq.com/product-preview/security-automation-pipelines/" >}}
+  Vulnerability Pipeline is in Preview. To enroll in the Preview for Mute rules, click <strong>Request Access</strong>.
+{{< /callout >}} 
+
+Configure mute rules to streamline security alerts by automatically filtering out non-urgent findings. This approach helps reduce noise from known false positives and accepted risks, allowing you to focus on addressing the most critical threats.
+
+## Create a mute rule
+
+1. On the [Vulnerability Pipeline][2] page, click **Add a New Rule** and select **Mute**.
+1. Enter a descriptive name for the rule, for example, Cloud Infrastructure Anomaly Warnings.
+1. Use the following boxes to configure the rule criteria:
+    - **Any of these types**: The types of findings that the rule should check for. Available types include **Misconfiguration**, **Attack Path**, **Identity Risk**, and **API Security Finding**.
+    - **Any of these tags or attributes**: The resource tags or attributes that must match for the rule to apply.
+1. To add severity criteria to the rule, click **Add Severity**.
+1. Specify the mute reason and duration:
+    - **Reason for muting**: The reason for muting the finding. Available reasons include **Duplicate**, **External solution**, **Pending fix**, **False positive**, **Internal solution**, **No fix available**, **Accepted risk**, and **Other**.
+    - **Rule expiration**: The date on which the rule expires. 
+    - **Further description for muting reason**: Optional box for additional details.
+1. Click **Save**.
+
+## Manage rule matching order
```

Comment:
This section looks to be informational, so I don't think that `manage` is an appropriate verb here - there aren't any instructions to manage the order. Remove the verb altogether?